docs/questions/visualizations/funnel.md

```diff
@@ -2,12 +2,87 @@
 ---
 
 # Funnel charts
 
-**Funnels** are commonly used in e-commerce or sales to visualize how many customers are present within each step of a checkout flow or sales cycle. At their most general, funnels show you values broken out by steps, and the percent decrease between each successive step. To create a funnel in Metabase, you'll need to have a table with at least two columns: one column that contains the metric you're interested in, and another that contains the funnel's steps.
-
-For example, I might have an Opportunities table, and I could create a question that gives me the number of sales leads broken out by a field that contains stages such as `Prospecting`, `Qualification`, `Proposal`, `Negotiation`, and `Closed`. In this example, the percentages shown along the x-axis tell you what percent of the total starting opportunities are still present at each subsequent step; so 18.89% of our total opportunities have made it all the way to being closed deals. The number below each percent is the actual value of the count at that step — in our example, the actual number of opportunities that are currently at each step. Together, these numbers help you figure out where you're losing your customers or users.
+Funnel charts visualize how a value is broken out by a series of steps, and the percent change between steps.
 
 ![Funnel](../images/funnel.png)
+
+Funnels are commonly used in e-commerce or sales to visualize how many customers are present within each step of a checkout flow or sales cycle.
+
+## Data shape for a funnel chart
+
+To create a funnel in Metabase, you'll need to have a table with at least two columns: one column that contains the funnel's steps, and another column contains the metric you're interested in, like the number of customers at this stage.
+
+Here's the data shape used to create the chart above:
+
+| Step          | Opportunities |
+| ------------- | ------------- |
+| Leads         | 3901          |
+| Qualification | 3714          |
+| Prospecting   | 3231          |
+| Proposal      | 2972          |
+| Negotiation   | 1588          |
+| Closed        | 737           |
+
+If you have more columns in your query results, you can select which columns should be used for the funnel steps and the metric in the [data settings](#funnel-chart-settings).
+
+By default, Metabase will present steps in the same order as they appear in the query result, but you can reorder or hide the steps in the [data settings](#funnel-chart-settings).
+
+## Build a query for a funnel chart
+
+To create a query with a result that has the shape required for a funnel chart, you'll probably need to summarize your data.
+
+If your (original, unaggregated) data already contains a field with the the step for every data point, you can build a simple query with a breakout by steps:
+
+![A query in the query builder used to build a funnel chart](../images/build-a-funnel-query.png)
```

Comment:
This is a large picture but I wanted to show both the unaggregated data and the aggregation.